src/main/java/com/redhat/devtools/lsp4ij/launching/ui/ShowInstructionDialog.java

```diff
@@ -41,18 +46,37 @@ protected ShowInstructionDialog(LanguageServerTemplate template, @Nullable Proje
         this.template = template;
         init();
     }
+
     @Override
     protected @Nullable JComponent createCenterPanel() {
         var descriptionBrowser = new DescriptionEditorPane();
         final var descriptionScrollPane = ScrollPaneFactory.createScrollPane(descriptionBrowser);
         descriptionScrollPane.setBorder(null);
-        DescriptionEditorPaneKt.readHTML(descriptionBrowser, template.getDescription());
+
+        // Convert Markdown to HTML
+        Parser parser = Parser.builder().build();

```

Comment:
Try to avoid using flexmark that we could remove it in the future since last version of IJ remove it because they use now their own Markdown parser. Please use our MarkdownConverter.